apps/portals/eliteportal/src/pages/HomePage.tsx

```diff
@@ -132,7 +133,7 @@ export default function HomePage() {
     </span>
   )
   return (
-    <>
+    <HomePageThemeProvider>
```

Comment:
Note that `theme` object on line 38 will now get the outer theme and not the home page theme. You could fix this by doing something like

```tsx
function HomePageInternal() {
  const theme = useTheme()
  // ...
}

function HomePage() {
  return (
    <HomePageThemeProvider>
      <HomePageInternal />
    </HomePageThemeProvider>
  )
}
```